### PR TITLE
Onboarding Screen Styles

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/101?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #83** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/83?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #84** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #87** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/87?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #92** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #93** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/93?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #94** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/94?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #95** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/95?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                    * **PR #96** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/96?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                      * **PR #99** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/99?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                        * **PR #101** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/101?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/101?utm_source=stack-comment).

File: packages/app/features/onboarding/screen.tsx
Comment (0xBigBoss) [inline packages/app/features/onboarding/screen.tsx:1]: thanks for starting on this one. Feel free to drop the "send demo" from `packages/playwright/tests/onboarding.logged-in.spec.ts`. I'm still tackling sending in my profiles PR.

**Review by 0xBigBoss — APPROVED** [review]
didn't visually check this one out, but LGTM

File: packages/app/features/onboarding/screen.tsx
Comment (youngkidwarrior) [reply packages/app/features/onboarding/screen.tsx:1]: Removed, double check for me that I understood you correctly and didn't delete anything important

File: packages/playwright/tests/onboarding.logged-in.spec.ts [hunk 1]
```diff
@@ -4,90 +4,10 @@
...
-  )
-  const cborAttObj = cbor.decodeAllSync(attestationObject)[0]
-  assert(!!cborAttObj, 'Missing cbor attestation object')
-  const { authData } = cborAttObj as Attestation
-  const { COSEPublicKey } = parseCredAuthData(authData)
-  assert(!!COSEPublicKey, 'Missing COSEPublicKey')
-  expect(webAuthnCred.public_key).toBe(`\\x${Buffer.from(COSEPublicKey).toString('hex')}`)
-
-  // sponsor the creation by setting the balance using anvil
-  await testBaseClient.setBalance({
```
Comment (0xBigBoss) [inline packages/playwright/tests/onboarding.logged-in.spec.ts:71]: Sorry, yeah you deleted too much. The "send" demo starts from this line and down. Perhaps, we move the checks above to the `OnboardingPage` fixture.

File: packages/app/features/onboarding/screen.tsx [hunk 2]
```diff
@@ -56,50 +62,108 @@ import {
...
+              <H2>Setup Passkey</H2>
+              <Paragraph>
+                Start by creating a Passkey below. Send uses passkeys to secure your account. Press
+                the button below to get started.
+              </Paragraph>
+            </YStack>
+            <CreateSendAccount />
+          </>
+        ) : (
+          <SendAccountUserOp />
```
Comment (0xBigBoss) [inline packages/app/features/onboarding/screen.tsx:150]: Let's drop this component entirely. This is the UI of the "send" demo I was referring to. It's no longer needed here.

Also drop the function here https://github.com/0xsend/sendapp/blob/99b109fb3aeb20ded846c52fa0583f560a145b88/packages/app/features/onboarding/screen.tsx#L179-L182

Then, for now, let's just show a paragraph or something similar. `Congratulations on opening your first Send Account! Let's get you sending, first, fund your account by sending ETH here ${address}`.

Could you screencap this `Congrats` part and post in design? We need to bring in Rit for that part of the design as well. 

You can add a `// @todo` here for above and just redirect to home page as well.

**Review by 0xBigBoss — CHANGES_REQUESTED** [review]

File: packages/app/features/onboarding/screen.tsx (hunk 2 above)
Comment (youngkidwarrior) [reply packages/app/features/onboarding/screen.tsx:150]: Can do

File: packages/playwright/tests/onboarding.logged-in.spec.ts (hunk 1 above)
Comment (youngkidwarrior) [reply packages/playwright/tests/onboarding.logged-in.spec.ts:71]: Which checks, the send demo stuff or the webAuthn stuff

File: packages/app/features/onboarding/screen.tsx (hunk 2 above)
Comment (youngkidwarrior) [reply packages/app/features/onboarding/screen.tsx:150]: @0xBigBoss See #107

File: packages/app/features/onboarding/screen.tsx [hunk 3]
```diff
@@ -269,7 +357,7 @@ function SendAccountUserOp() {
     <YStack space="$4">
       <H2>Send it</H2>
       <Label htmlFor="sendTo">Sending to:</Label>
-      <Input id="sendTo" value={receiverAccount.address} />
+      <Input id="sendTo" aria-label="sendTo" value={receiverAccount.address} />
```
Comment (0xBigBoss) [inline packages/app/features/onboarding/screen.tsx:360]: I think the x-platform way is [accessibilityLabel](https://reactnative.dev/docs/accessibility#accessibilitylabel), but maybe tamagui sets this up for us automatically. 🤷

File: packages/playwright/tests/onboarding.logged-in.spec.ts (hunk 1 above)
Comment (0xBigBoss) [reply packages/playwright/tests/onboarding.logged-in.spec.ts:71]: Sorry, I wasn't more specific. This changset looks good. I don't think we can refactor the `expect`'s above this line.

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 8, 1:24 PM EST**: [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/101) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**